protobuf/meshnet/service.proto

```diff
@@ -38,6 +38,8 @@ service Meshnet {
 	rpc GetPeers(Empty) returns (GetPeersResponse);
 	// RemovePeer removes a peer from the meshnet
 	rpc RemovePeer(UpdatePeerRequest) returns (RemovePeerResponse);
+	// RenamePeer set or clear a nickname for a meshnet peer
+	rpc RenamePeer(RenamePeerRequest) returns (RenamePeerResponse);
```

Comment:
Semantically it would be more clear to have 2 methods: `Set` and `Remove` as by API it is `peer add nickname` and `peer remove nickname` but not `peer rename`

Reply:
I've tried to have 2 separate function when setting nickname for own machine, but in the end one of them calls the other one. Because of this I'm not sure if it brings anything to have 2 calls.
Instead I was thinking to change it to `ChangePeerNickname` or something similar?

Reply:
changed to `ChangePeerNickname`